distributed/tests/py3_test_asyncio.py

```diff
@@ -241,7 +247,6 @@ def f(x, y=10):
 async def test_asyncio_cancel_tuple_key():
     async with AioClient(processes=False) as c:
         x = c.submit(inc, 1, key=('x', 0, 1))
-        await x
```

Comment:
Hi Matt, I removed this as an experiment to debug the timeout error in Travis. I had a theory we shouldn't await a Future, then cancel it, which turned out not to be relevant. I'm still trying to figure out the cause of the error.